community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -395,6 +395,14 @@ private static String defaultPageCacheMemory()
     @Description("Path of the logs directory")
     public static final Setting<File> logs_directory = setting("dbms.directories.logs", PATH, NO_DEFAULT);
 
+    @Description("Enable GC Logging")
+    public static final Setting<Boolean> gc_logging_enabled = setting("dbms.logs.gc.enabled", BOOLEAN, FALSE);
+
+    @Description("GC Logging Options")
+    public static final Setting<String> gc_logging_options = setting("dbms.logs.gc.options", STRING, "" +
+            "-XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintGCApplicationStoppedTime " +
+            "-XX:+PrintPromotionFailure -XX:+PrintTenuringDistribution");
+
```

Comment:
All these GC options should be in `ServerSettings` as they are not applicable to embedded.